support/Chart.yaml

```
@@ -29,3 +29,8 @@ dependencies:
     version: 9.10.3
     repository: https://kubernetes.github.io/autoscaler
     condition: cluster-autoscaler.enabled
+
+  # In-cluster NFS server provisioner
+  - name: nfs-server-provisioner
+    version: 1.3.1
+    repository: https://kvaps.github.io/charts
```

Probably make sense to have this be something one can opt-in/opt-out from when installing the support chart and all its dependencies.

```suggestion
    repository: https://kvaps.github.io/charts
    condition: nfs-server-provisioner.enabled
```